mockgcp/mockcompute/globalforwardingrulesv1.go

```diff
@@ -125,6 +125,12 @@ func (s *GlobalForwardingRulesV1) Insert(ctx context.Context, req *pb.InsertGlob
 		}
 	}
 
+	// network field is only used for global internal load balancing.
+	// If neither subnetwork nor network field is specified, the default network will be used.
+	if obj.Network == nil && obj.Subnetwork == nil && obj.LoadBalancingScheme != nil && *obj.LoadBalancingScheme != "EXTERNAL" {
```

Comment:
for my own learning, could I get a pointer to the api here?

Reply:
Do you mean comparing pointers? that will cause issues since the pointers are different, I want to check the value here... Not sure if there's a better way to achieve that.